actioncable/app/assets/javascripts/action_cable.coffee.erb

```diff
@@ -1,3 +1,4 @@
+#= export ActionCable
```

Comment:
No. Unknown Sprockets directives are ignored so without Blade the default behavior applies (setting `window.ActionCable`).